br/pkg/metautil/metafile.go

```diff
@@ -427,19 +427,22 @@ func (op AppendOp) name() string {
 }
 
 // appends item to MetaFile
-func (op AppendOp) appendFile(a *backuppb.MetaFile, b interface{}) (size int, itemCount int) {
+func (op AppendOp) appendFile(a *backuppb.MetaFile, b interface{}) (sstFilesSize int, size int, itemCount int) {
 	switch op {
 	case AppendMetaFile:
-		a.MetaFiles = append(a.MetaFiles, b.(*backuppb.File))
-		size += int(b.(*backuppb.File).Size_)
+		metaFile := b.(*backuppb.File)
+		a.MetaFiles = append(a.MetaFiles, metaFile)
+		size += metaFile.Size()
+		sstFilesSize += int(metaFile.Size_)
 		itemCount++
 	case AppendDataFile:
 		// receive a batch of file because we need write and default sst are adjacent.
 		files := b.([]*backuppb.File)
 		a.DataFiles = append(a.DataFiles, files...)
 		for _, f := range files {
 			itemCount++
-			size += int(f.Size_)
+			size += f.Size()
+			sstFilesSize += int(f.Size_)
```

Comment:
Maybe `sstFilesSize` calls `dataFileSize` better? Because the op name is `AppendDataFile`

Reply:
Done.